CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ description: |-
 ---
 
 # VKCS Provider's changelog
+
+#### v0.7.3 (unreleased)
+- Add a warning when creating a block_device with incorrect use of the delete_on_termination parameter
```

Comment:
I think writing a changelog for this is a bit overkill, it doesn't change the user experience noticeably.

Reply:
Can i skip the ci/cd checks because it requires changes in changelog?

Reply:
done